resolver/resolver.go

```diff
@@ -389,6 +415,14 @@ func (res *Resolver) handleSRV(rs *records.RecordGenerator, name string, m, r *d
 			}
 			m.Extra = append(m.Extra, aRR)
 			added[host] = struct{}{}
+		} else if aaaa, ok := rs.AAAAs.First(host); ok {
```

Comment:
why only append these if ipv4 isn't found? why can't we return both? (or is that not kosher?)

Reply:
You're correct, this was an oversight